src/xSTIR/cSTIR/cstir.cpp

```diff
@@ -989,6 +989,21 @@ void* cSTIR_getImageVoxelSizes(const void* ptr_im, size_t ptr_vs)
 	CATCH;
 }
 
+extern "C"
+void* cSTIR_getImageTransformMatrix(const void* ptr_im, size_t ptr_md)
+{
+	try {
+		PETImageData& id = objectFromHandle<PETImageData>(ptr_im);
+		float* data = (float*)ptr_md;
```

Comment:
some weird syntax? Does this actually allocate anything? Seems very dangerous to me

Reply:
this is in all of the interface